packages/components/checkbox/src/Checkbox.doc.mdx

```diff
@@ -24,108 +25,151 @@ import { Checkbox, CheckboxGroup } from '@spark-ui/checkbox'
 
-## Usage
+<Canvas of={stories.Disabled} />
 
-<Canvas of={CheckboxStories.Group} />
+### Icon
 
-## Using a FormField
+Use `icon` prop to change the default checked icon.
 
-If you need to provide more context about the Checkbox (for example, a helper message or an error message) the FormField component could be used. See [documentation](./?path=/docs/experimental-formfield--docs) for more details.
+<Canvas of={stories.Icon} />
 
-<Canvas of={CheckboxStories.GroupImproved} />
+### Intent
 
-## Intent
+Use `intent` prop to change the color of the checkbox.
 
-Use `intent` prop to change the color of all the checkboxes within the group.
+**Note: set `intent` on the `CheckboxGroup` if you want to apply the same intent to every checkbox within a group**
 
-<Canvas of={CheckboxStories.GroupIntent} />
+<Canvas of={stories.Intent} />
 
-## Orientation
+### Orientation
 
-Use `orientation` prop to set the orientation of the checkbox group which could be `vertical` or `horizontal`.
+**Only for `CheckboxGroup`**
 
-<Canvas of={CheckboxStories.GroupOrientation} />
-
-## Uncontrolled
+Use `orientation` prop to set the orientation of the checkbox group which could be `vertical` or `horizontal`.
 
-The `defaultValue` prop can be used to set the default state of the group. Optionally `onChange` prop can be used to get the current state of the checkbox group.
+<Canvas of={stories.GroupOrientation} />
 
-<Canvas of={CheckboxStories.GroupUncontrolled} />
+## Advanced usage
 
-## Controlled
+### Indeterminate
 
-Use `value` and `onChange` props to control the state of the checkbox.
+You can set the checkbox to indeterminate by taking control of its state.
 
-<Canvas of={CheckboxStories.GroupControlled} />
+<Canvas of={stories.Indeterminate} />
 
-## Custom implementation
+### Custom implementation
 
 Below is an implementation of a custom checkbox that might provide guidance if you need to implement something custom as well.
 
 When implementing a custom checkbox, it is important to be careful to properly handle **accessibility (a11y) requirements**, as a11y considerations will not be handled behind the scenes as they would be when using a regular checkbox implementation
 
-<Canvas of={CheckboxStories.CustomImplementation} />
+<Canvas of={stories.CustomImplementation} />
 
-## Invisible checkbox
+### Invisible checkbox
 
 We can go even further by completely hiding the checkbox using our [`VisuallyHidden`](?path=/docs/components-visuallyhidden--docs#visuallyhidden) component. This way, you end up with a component that shares all the intrinsic behaviors of a checkbox, but does not look like one.
 
 When implementing a custom checkbox, it is important to be careful to properly handle **accessibility (a11y) requirements**, as a11y considerations will not be handled behind the scenes as they would be when using a regular checkbox implementation
 
-<Canvas of={CheckboxStories.InvisibleCheckbox} />
+<Canvas of={stories.InvisibleCheckbox} />
+
+## FormField
```

Comment:
Is the same wording that we used for the other stories ?

Reply:
You are right, it should be "Form field"